plugins/flytekit-greatexpectations/flytekitplugins/great_expectations/schema.py

```diff
@@ -66,7 +66,7 @@ class GreatExpectationsType(object):
     """
 
     @classmethod
-    def config(cls) -> Tuple[Type, Type[GreatExpectationsFlyteConfig]]:
+    def config(cls) -> Tuple[Type[str], GreatExpectationsFlyteConfig]:
```

Comment:
This cannot be Type[str]. config() returns a type and the config. The type can be str, FlyteFile or FlyteSchema.